### PR TITLE
makes my site responsive

### LINKED ISSUE #265: make my site responsive
currently, my site is not responsive, we need to make it responsive.

Images for reference

image
image
image

### PR DESCRIPTION
**Pull request by swapnadeepmohapatra**
This fixes #265 

image
image
image

### CODE REVIEW

**Comment by vinayak-trivedi** [comment]
hey, @swapnadeepmohapatra the task was regarding the whole my site, not just the identity service page. Check the [issue](https://github.com/Real-Dev-Squad/website-my/issues/265)

**Comment by swapnadeepmohapatra** [comment]
Hey @vinayak-trivedi I have made changes to the PR. Can you please check them?

File: app/styles/identity.css [hunk 1]
```diff
@@ -1,36 +1,24 @@
...
 .input-div {
-    width: 30%;
-}
-
-.input-url {
-    height: 2rem;
-    margin-top: 25px;
-    width: 60%;
-}
-
```
Comment (vinayak-trivedi) [inline app/styles/identity.css:25]: why is this removed?

**Comment by singh3069** [comment]
- ### CSS is breaking at smaller devices.
image

- ###  the check box is at the edge of the page make it more to the left.
image

- ### and add some space in between these two
image

**Comment by swapnadeepmohapatra** [comment]
- **Moved the check box to more left**

image

- **Added some space in between these two**

image

**Comment by swapnadeepmohapatra** [comment]
- Decreased the font size of notifications in mobile view. 
- Changed the notifications title from H3 to P. 

image

File: app/styles/notification.css [hunk 2]
```diff
@@ -84,6 +84,7 @@
 .text-title {
   font-weight: bold;
   font-size: 20px;
+  padding-bottom: 4px;
```
Comment (rajat-mehra05) [inline app/styles/notification.css:87]: Instead of using `padding-bottom` here, use `gap` on parent container.
That's a good practice.

**Review by rajat-mehra05 — CHANGES_REQUESTED** [review]

**Comment by swapnadeepmohapatra** [comment]
image

Have fixed the spacing of notification heading

File: app/styles/notification.css (hunk 2 above)
Comment (swapnadeepmohapatra) [reply app/styles/notification.css:87]: The parent is not a flex. And when it is changed to flex, then the CSS is breaking. 

image

**Comment by rajat-mehra05** [comment]
> image
> 
> Have fixed the spacing of notification heading

Looks good now. Thanks. Please resolve above conversation once the change is done.

File: app/styles/notification.css (hunk 2 above)
Comment (swapnadeepmohapatra) [reply app/styles/notification.css:87]: > The parent is not a flex. And when it is changed to flex, then the CSS is breaking.
> 
> image

@rajat-mehra05

Comment (rajat-mehra05) [reply app/styles/notification.css:87]: Okay got it.

File: app/styles/app.css [hunk 3]
```diff
@@ -58,3 +55,11 @@ body {
   align-items: center;
   font-size: 28px;
 }
+
+
+@media (max-width: 460px) {
+  .footer {
+    padding: 8px 8px 16px 8px;
```
Comment (vinayak-trivedi) [inline app/styles/app.css:62]: can this be changed to `8px 8px 16px`, because you are providing the same padding for left and right

File: app/styles/identity.css [hunk 4]
```diff
@@ -39,20 +27,42 @@
...
 }
 
-.identity-label {
-    margin-top: 30px;
-    text-align: center;
+.copy-button {
+    margin-left: 0.75rem;
+    background-color: wheat;
+    color: #e30062;
+    border: 2px solid #e30062;
```
Comment (vinayak-trivedi) [inline app/styles/identity.css:34]: can we put this color in a css variable and then use it here?

File: app/styles/notification.css [hunk 5]
```diff
@@ -1,62 +1,103 @@
...
 .tip-box-alert {
-    color: #212121;
-    background: #FFF8E1;
+  color: #212121;
+  background: #fff8e1;
 }
 
 .tip-icon-alert {
-    background: #FF6F00;
+  background: #ff6f00;
```
Comment (vinayak-trivedi) [inline app/styles/notification.css:70]: color used here can be put in a css variable and then you can use it there

**Review by vinayak-trivedi — DISMISSED** [review]
LGTM

File: app/styles/identity.css [hunk 6]
```diff
@@ -1,58 +1,74 @@
...
+    gap: 1rem;
+}
+
+.identity-check-label {
+    text-align: center;
+    display: flex;
+    align-items: center;
+    flex-direction: row;
+    gap: 1rem;
+}
```
Comment (rohan09-raj) [inline app/styles/identity.css:68]: We can merge these CSS blocks. Also, the flex-direction value is  `row` by default. No need to explicitly provide it

```suggestion
.identity-label,
.identity-check-label {
    text-align: center;
    display: flex;
    align-items: center;
    gap: 1rem;
}
```

File: app/templates/notifications.hbs [hunk 7]
```diff
@@ -1,3 +1,5 @@
+<div class="notifications">
 {{#each @model as |post|}}
   <NotificationCard @title={{post.title}} @description={{post.description}} />
 {{/each}}
+</div>
```
Comment (rohan09-raj) [inline app/templates/notifications.hbs:5]: Can we please have 2 tab spacing inside the div?
```suggestion
<div class="notifications">
  {{#each @model as |post|}}
    <NotificationCard @title={{post.title}} @description={{post.description}} />
  {{/each}}
</div>
```

File: app/styles/app.css [hunk 8]
```diff
@@ -58,3 +55,10 @@ body {
   align-items: center;
   font-size: 28px;
 }
+
+@media (max-width: 460px) {
+  .footer {
+    padding: 8px 8px 16px;
+    font-size: 14px;
```
Comment (rohan09-raj) [inline app/styles/app.css:62]: Can we please follow `rem`?

File: app/styles/local-navbar.css [hunk 9]
```diff
@@ -17,3 +17,13 @@
...
   color: #e30062;
 }
+
+@media screen and (max-width: 371px) {
+  .local-navbar__link a {
+    padding: 5px;
+  }
+
+  .local-navbar {
+    font-size: 12px;
```
Comment (rohan09-raj) [inline app/styles/local-navbar.css:27]: Can we please follow `rem`?

File: app/styles/notification.css [hunk 10]
```diff
@@ -1,62 +1,106 @@
...
+  }
+}
+
+.text-title {
+  font-weight: bold;
+  font-size: 20px;
+}
+
+.text-description {
+  font-size: 16px;
```
Comment (rohan09-raj) [inline app/styles/notification.css:94]: Can we please follow `rem` for font sizes?

File: app/styles/notification.css [hunk 11]
```diff
@@ -1,62 +1,106 @@
...
+}
+
+@media (max-width: 480px) {
+  .text-title {
+    font-weight: bold;
+    font-size: 16px;
+  }
+
+  .text-description {
+    font-size: 12px;
```
Comment (rohan09-raj) [inline app/styles/notification.css:104]: Can we please follow `rem` for font size?

**Review by rohan09-raj — APPROVED** [review]
Thanks for making the my-site responsive. Approving the Pull Request 💯 🔥

**Comment by swapnadeepmohapatra** [comment]
Thank you @rohan09-raj @vinayak-trivedi @singh3069 @rajat-mehra05
